Soomla/Assets/Soomla/Scripts/Profile/Config/ProfileSettings.cs

```diff
@@ -53,7 +53,6 @@ static ProfileSettings()
 			additionalDependFiles.Add("Assets/Plugins/Android/Soomla/libs/twitter4j-core-4.0.2.jar");
 			additionalDependFiles.Add("Assets/Plugins/iOS/Soomla/libSoomlaiOSProfileGoogle.a");
 			additionalDependFiles.Add("Assets/Plugins/Android/Soomla/libs/AndroidProfileGoogle.jar");
-			additionalDependFiles.Add("Assets/Plugins/Android/Soomla/libs/google-play-services_lib");
```

Comment:
@Blinkq, why it was deleted here? Shouldn't we remove it when Soomla is removing?